rebasehelper/application.py

```diff
@@ -413,6 +421,39 @@ def patch_sources(self, sources):
         self.rebase_spec_file.write_updated_patches(self.rebased_patches)
         results_store.set_patches_results(self.rebased_patches)
 
+    def generate_patch(self):
+        """
+        Generates patch to the results_dir containing all needed changes for
+        the rebased package version
+        """
+        # Delete removed patches from rebased_sources_dir from git
+        for removed_patch in self.rebase_spec_file.removed_patches:
+            self.git_helper.command_rm(os.path.join(self.rebased_sources_dir, removed_patch))
+
+        # Fix paths in rebase_spec_file to patches to current directory
+        for index, line in enumerate(self.rebase_spec_file.spec_content):
+            mod_line = re.sub(settings.REBASE_HELPER_REBASED_SOURCES_DIR + '/', '', line)
+            self.rebase_spec_file.spec_content[index] = mod_line
+        self.rebase_spec_file.save()
+
+        # Generate patch
+        self.git_helper.command_add_files('.')
+        self.git_helper.command_commit(message='Get difference between versions')
```

Comment:
I think it would be better if the commit message would be the same as changelog entry.